Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormLawRegistered.cshtml

```diff
@@ -23,7 +23,9 @@
             <form class="form" method="post">
                 <div class="govuk-form-group @(lawRegisteredHasError ? "govuk-form-group--error": "")">
                     <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-                        <h1 class="govuk-heading-l">@ViewData["Title"]</h1>
+                        <h1 class="govuk-heading-l">
+                            <label class="govuk-label govuk-label--l" for="lawRegistered-text">@ViewData["Title"]</label>
+                        </h1>
```

Comment:
@rmohammed-goaco Please could you undo just this one actually. If you look at the following PR you'll see that this page has been completely reworked and the issue is resolved, but differently to this...

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/pull/1475/files#diff-b549516de48710ee9a7d40a1da7fb75ac09c271935304486949b62aabd4ebaf4R21

Reply:
Thanks!